workflow/rules/hiphase.smk

```diff
@@ -0,0 +1,62 @@
+__author__ = "Padraic Corcoran"
+__copyright__ = "Copyright 2024, Padraic Corcoran"
+__email__ = "padraic.corcoran@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule hiphase:
```

Comment:
Nice solution, like a wrapper but in the rule

Reply:
Thanks!